planet/src/main/java/com/google/planet/data/TimeRange.java

```diff
@@ -15,9 +15,9 @@
 package com.google.planet.data;
 
 public final class TimeRange {
-    public static final int START_OF_DAY = getTimeInMinutes(8, 0);
-    public static final int END_OF_DAY = getTimeInMinutes(19, 59);
-    public static final TimeRange WHOLE_DAY = new TimeRange(getTimeInMinutes(8,0), 12 * 60);
+    public static final int START_OF_DAY = getTimeInMinutes(0, 0);
+    public static final int END_OF_DAY = getTimeInMinutes(23, 59);
+    public static final TimeRange WHOLE_DAY = new TimeRange(getTimeInMinutes(0,0), 24 * 60);
```

Comment:
Why have you decided to switch back to 00:00 -> 23:59?

Reply:
It was 8 am to 7pm before, but then some restaurants have opening hours till very late (11pm etc.) and the timerange class will return an error because of that

Reply:
I see! okay